test/models/concerns/effective_devise_user_test.rb

```
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+class EffectiveDeviseUserTest < ActiveSupport::TestCase
+
+  test 'email uniqueness' do
+    User.create(first_name: "A", last_name: "User", email: "user@mail.com",  alternate_email: "alter@mail.com")
```

could you use `User.create!` here so it will fail loudly if it fails